.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,33 @@
+name: Fuzz CI
+on:
+  pull_request:
+    branches: ["master", "dev"]
```

Comment:
We always open PRs against `dev`, is there a reason to include `master` also?

Reply:
No specific reason. I have removed the master.